CONTRIBUTOR_ROLES.md

```diff
@@ -110,12 +109,9 @@ To become a project Maintainer, you must meet the following requirements:
 * You have directly contributed to the project through implementation and/or review.
 * You have been sponsored by two Maintainers.
 
-If you meet the requirements, nominate yourself to become a Maintainer by sending an email to the Maintainers with your candidacy. You must:
+If you meet the requirements, have one of your sponsors open an issue in the relevant repository with the provided nomination template.
```

Comment:
```suggestion
One of the sponsors should open an issue in the relevant repository to provide a nomination. The issue should use the provided nomination template if one exists in that repo.
```

Reply:
Appending this text as well: `If one does not exist, the sponsor should create a template prior to creating the nomination.`